src/app/utils/textarea-caret.ts

```diff
@@ -78,7 +78,7 @@ export default function getCaretCoordinates(
   style.visibility = 'hidden'; // not 'display: none' because we want rendering
 
   // Transfer the element's properties to the div
-  properties.forEach(function (prop) {
+  for (const prop of properties) {
```

Comment:
This code was copied from [a dependency](https://github.com/component/textarea-caret-position/blob/master/index.js) to make the `// DIM: ...` change below, so personally I wouldn't apply all our style lints to it.